qmart.css

```diff
@@ -0,0 +1,109 @@
+@import url(https://quik-ui.netlify.app/style.css);
+
+@import url(./styles/home.css);
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --gray-200: #f5f5f5;
+  --gray-300: #eeeeee;
+  --primary: #0e161a;
+  --secondary-color: #ff3b3b;
+  --secondary-light: hsla(0, 100%, 62%, 0.8);
+  --blue: #0a66c2;
+}
+
+.q-main-container {
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  min-height: 100vh;
+}
+
+.q-nav {
+  width: 100%;
+  padding: var(--spacing-1) var(--spacing-2);
+  background-color: var(--white);
+}
+
+.q-logo {
+  color: var(--secondary-color);
+}
+
+.q-search-container span {
+  color: var(--gray-color);
+  font-weight: 400;
+  left: var(--spacing-1);
+}
+
+.q-search-container .input {
+  width: 35rem;
+  padding-left: var(--spacing-3);
+  border-radius: 5px;
+  border-color: var(--primary);
+}
+
+.q-nav-options > *:not(:nth-child(1)) {
+  margin-left: var(--spacing-2);
+}
+
+.q-nav-options .q-nav-login {
+  background-color: var(--secondary-light);
+  padding: var(--spacing-1) var(--spacing-2);
```

Comment:
padding 2 times

Reply:
Yes, For --> top and bottom I used - 'spacing-1'
            -->  left and right I used - 'spacing-2'